src/shogun/labels/Labels.cpp

```diff
@@ -90,7 +100,33 @@ void CLabels::set_values(SGVector<float64_t> values)
 	m_current_values = values;
 }
 
-SGVector<float64_t> CLabels::get_values()
+SGVector<float64_t> CLabels::get_values() const
 {
 	return m_current_values;
 }
+
+Some<CBinaryLabels> CLabels::as_binary() const
```

Comment:
mentioned conversion method to convert label types into each other

Reply:
see above regarding move/rvalue